app/src/main/java/com/github/sdpsharelook/storage/IRepository.kt

```diff
@@ -12,7 +12,7 @@ interface IRepository<T> {
      * @param name identifier of entity
      * @return [Flow] of changes in the database at [name]
      */
-    fun flow(name: String = "test"): Flow<Result<T?>>
+    fun flow(name: String = "test"): Flow<Result<@JvmSuppressWildcards T?>>
```

Comment:
This is unnecessary, all you need is to put that in the generic type of the thing you're trying to hilt-Bind.
Will fix later